product_docs/docs/efm/4/04_configuring_efm/01_cluster_properties.mdx

```diff
@@ -503,6 +506,11 @@ Use the `node.timeout` property to specify the number of seconds for an agent to
 node.timeout=50
 ```
 
+!!! Summary/comparison of timeout properties
+    - The `local.*` properties are for failure detection of an agent's local database.
+    - The `node.timeout` property is for failure detection of other nodes.
+    - The `remote.timeout` property limits how long agents wait for responses from other agents.
+
```

Comment:
Just trying to get this fully understood in my head.

During some cluster events, the agents communicate to each other. the agents try and contact each other for `remote.timeout` seconds.

After that if they can't communicate, what happens? They then ping the node (server?) that hosts the agent, to see if the host is online, and they ping that for `node.timeout`. If no response, they declare that host out of the cluster?

Reply:
It depends on what the question is. If you have specific cases you want to ask about, feel free to do so in slack.